microsoft-edge/develop-web-microsoft-edge.md

```diff
@@ -101,7 +101,7 @@ See [Test and automation in Microsoft Edge](test-and-automation/test-and-automat
 
 
 <!-- ====================================================================== -->
-## Web platform
+## Development tips for Microsoft Edge
 
 Considerations for developing websites and products for the web platform include the following:
```

Comment:
```suggestion
There are many tips and tricks to be aware of when building on the web platform, which include: 
```

Reply:
fixed locally, trying (possibly more natural/transparent):
There are many tips and tricks to be aware of when building on the web platform, including: